contracts/security-token/UnrestrictedTransferAgent.sol

```diff
@@ -8,6 +8,14 @@ contract UnrestrictedTransferAgent is SecurityTransferAgent {
   function UnrestrictedTransferAgent() {
   }
 
+  /**
+   * @dev Checking if transfer can happen, and if so, what is the right amount
+   *
+   * @param from The account sending the tokens
+   * @param to The account receiving the tokens
+   * @param value The indended amount
+   * @return The actual amount permitted
+   */
   function verify(address from, address to, uint256 value) public returns (uint256 newValue) {
     return value;
```

Comment:
is thus code just for mock up or do we plan to add more checks here. 
```
/* We invoke UnrestrictedTransferAgent here, because whatever it wants to do
         (like emit an event), we want to do too. */
```
asking because I read this comment near `super.verify(from, to, value);` in `RestrictedTransferAgent`

Reply:
The initial plan was to give some room for development, but we decided to ditch events here :slightly_smiling_face: